File: java/inHere-java/src/main/java/com/inHere/entity/AskReply.java
package com.inHere.entity;

import java.util.Date;
import java.util.HashMap;
import java.util.Map;

/**
 * 吐槽+问答
 * 
 * @author lwh
 *
 */
public class AskReply {

	private Integer id;
	private Integer extType;
	private String extData;
	private Integer labelId;
	private String title;
	private String content;
	private String photos;
	private Integer commentNum;
	private String userId;
	private Date createTime;
	private Date updateTime;
	private String praise;
	private String low;

	private Label label; // 关联标签

	private Comment bestReply; // 最火的评论或答案

	public Comment getBestReply() {
		return bestReply;
	}

	public void setBestReply(Comment bestReply) {
		this.bestReply = bestReply;
	}

	public Label getLabel() {
		return label;
	}

	public void setLabel(Label label) {
		this.label = label;
	}

	public String getContent() {
		return content;
	}

	public void setContent(String content) {
		this.content = content;
	}

	public Integer getId() {
		return id;
	}

	public void setId(Integer id) {
		this.id = id;
	}

	public Integer getExtType() {
		return extType;
	}

	public void setExtType(Integer extType) {
		this.extType = extType;
	}

	public String getExtData() {
		return extData;
	}

	public void setExtData(String extData) {
		this.extData = extData == null ? null : extData.trim();
	}

	public Integer getLabelId() {
		return labelId;
	}

	public void setLabelId(Integer labelId) {
		this.labelId = labelId;
	}

	public String getTitle() {
		return title;
	}

	public void setTitle(String title) {
		this.title = title == null ? null : title.trim();
	}

	public String getPhotos() {
		return photos;
	}

	public void setPhotos(String photos) {
		this.photos = photos == null ? null : photos.trim();
	}

	public Integer getCommentNum() {
		return commentNum;
	}

	public void setCommentNum(Integer commentNum) {
		this.commentNum = commentNum;
	}

	public String getUserId() {
		return userId;
	}

	public void setUserId(String userId) {
		this.userId = userId == null ? null : userId.trim();
	}

	public Date getCreateTime() {
		return createTime;
	}

	public void setCreateTime(Date createTime) {
		this.createTime = createTime;
	}

	public Date getUpdateTime() {
		return updateTime;
	}

	public void setUpdateTime(Date updateTime) {
		this.updateTime = updateTime;
	}

	public String getPraise() {
		return praise;
	}

	public void setPraise(String praise) {
		this.praise = praise == null ? null : praise.trim();
	}

	public String getLow() {
		return low;
	}

	public void setLow(String low) {
		this.low = low == null ? null : low.trim();
	}

	@Override
	public String toString() {
		return "AskReply [id=" + id + ", extType=" + extType + ", extData=" + extData + ", labelId=" + labelId
				+ ", title=" + title + ", content=" + content + ", photos=" + photos + ", commentNum=" + commentNum
				+ ", userId=" + userId + ", createTime=" + createTime + ", updateTime=" + updateTime + ", praise="
				+ praise + ", low=" + low + ", label=" + label + ", bestReply=" + bestReply + "]";
	}

	public Map<String, Object> toMap() {
		Map<String, Object> map = new HashMap<String, Object>();
		map.put("", "");
		return map;
	}

}

File: client/app/src/api/fake/ask_reply.js
/**
 * Created by anzer on 2016/9/9.
 */
import  moment from 'moment'
let label={
  10:[
    {
      id: 1, //标签编号
      name: "你们饭堂有什么奇葩的菜式"
    },
    {
      id:2, //标签编号
      name: "说说哪些让你睡着的老师"
    },
    {
      id: 3, //标签编号
      name: "一起来吐槽女朋友"
    },
    {
      id: 4, //标签编号
      name: "一起来吐槽男票"
    }
  ],
  11:[
    {
      id: 1, //标签编号
      name: "小道消息"
    },
    {
      id:2, //标签编号
      name: "老师"
    },
    {
      id: 3, //标签编号
      name: "饭堂"
    },
    {
      id: 4, //标签编号
      name: "情感"
    }
  ],
  12:[
    {
      id: 1, //标签编号
      name: "宿舍"
    },
    {
      id:2, //标签编号
      name: "新生入学"
    },
    {
      id: 3, //标签编号
      name: "补考"
    },
    {
      id: 4, //标签编号
      name: "学习"
    }
  ]
};
let data=[{
  id: 1, //实体id
  ext_type: 10, // 类别，10-校内吐槽，11-校外吐槽，12-问答
  ext_data: {},  //私有属性
  user_id: "用户编号",
  title: "标题",
  photos: [
    {
      min: xxxx, // 小图数据base64
      src: url, // 大图url
      w: 100,	  //长
      h: 100    //高
    },
    // .........四张
  ],
  label_id: 2, //标签编号
  label_name: "标签名称",
  create_time: 223232312334, // 创建时间
  comment_num: 233, //评论数
  praise: 88  //赞数目
},
  {
    id: 1, //实体id
    ext_type: 12, // 类别，10-校内吐槽，11-校外吐槽，12-问答
    ext_data: {   //私有属性
      follow: 0, //关注帖子, 0-未，1-已
    },
    user_id: "用户编号",
    title: "标题",
    photos: [],
    label_id: 2, //标签编号
    label_name: "标签名称",
    create_time: 223232312334, // 创建时间
    comment_num: 233, //评论数
    praise: 88  //赞数目
  }];
